website/docs/registry/faq.mdx

```diff
@@ -0,0 +1,124 @@
+
+Each version of an integration published in the Terraform Registry has a version identifier, which is a string like `1.2.3`. You create the version identifier when you create a tag (for modules and policy libraries) or a release (for providers) in the GitHub repository where the integration's code lives.
+
+The Registry requires that these version identifiers are [SemVer compliant](https://semver.org/). Versions are sorted using these identifiers, following [standard SemVer rules for precedence](https://semver.org/#spec-item-11). The version identifier with the greatest precedence is always listed as latest by the Terraform Registry.
+
+### Why are prerelease versions listed as the latest version of my provider, module, or policy library?
+
+The Terraform Registry's version sorting is strictly SemVer compliant (see above). This means that "latest" in the Registry can be a prerelease version, if the prerelease's version identifier has the greatest precedence.
+
+For example, if your provider has a published version 1 (i.e. `1.0.0`), and then you create a new release for a version 2 prerelease (e.g. `2.0.0-preview`), the version 2 prerelease will be listed as the latest version in the Terraform Registry.
+
+Please note that terraform itself will never select a prerelease version during a `plan` or `apply`, unless the configuration files explicitly request one [using an exact version constraint](/terraform/language/expressions/version-constraints#version-constraint-behavior).
+
+### How do I change the GPG key that I use to sign my provider?
+
+The public GPG keys that you upload to the registry serve two purposes:
+
+1. When you publish a new version of a provider, the Registry can use this GPG key to verify the signature file you created. This lets the Registry confirm that the binary files were, in fact, compiled by you.
+2. When someone download a version of your provider from the Registry, they can verify that the binary files that they download have not been tampered with since you built them and generated the signature file.
+
+If you have previously used a certain GPG key to sign and publish providers, and you now wish to use a new one, you need only upload the new public GPG key, and use its private key to sign provider releases from now on.
+
+You do not need to remove any old GPG keys. Leaving the old key in place allows consumers of old versions of your provider to continue using those versions.
+
+### How do I remove a public GPG key from the Registry that has been used for publishing a provider?
+
+Generally, you should not ever remove a public GPG key from the Registry. Because of this, we don't provide a way for you to do it yourself.
+
+If you were to remove a public GPG key that has been used to sign previous versions of a provider, consumers would be unable to use those older versions, because they would be unable to verify the authenticity of the binary file they downloaded.
```

Comment:
```suggestion
If you removed a public GPG key that you used to sign previous versions of a provider, consumers would be unable to use those versions because they could not verify the authenticity of the binary file they downloaded.
```